editor/editor_data.cpp

```diff
@@ -364,6 +364,14 @@ void EditorData::notify_edited_scene_changed() {
 	}
 }
 
+void EditorData::notify_resource_saved(const Object *p_resource) {
```

Comment:
If this is for resources only, isn't better for the argument to be a `Ref<Resource>` instead of `Object*`?